apps/right-to-rent-check/views/pdf.html

```diff
@@ -1,8 +1,8 @@
 {{<govuk-template}}
   {{$bodyClasses}}pdf{{/bodyClasses}}
   {{$head}}
-    {{> partials-head}}
     <style>
+      {{{css}}}
       .pdf #global-header,
       .pdf footer,
       .pdf #declaration-section,
```

Comment:
Could this css be added to `css/app.css`?

Reply:
It could, but since it's needed exactly nowhere else, then it would just add page-weight for little real benefit.